File: win-prog/ProgWinVB/Files and Streams/StreamWriterDemo/StreamWriterDemo.vb
'-------------------------------------------------
' StreamWriterDemo.vb (c) 2002 by Charles Petzold
'-------------------------------------------------
Imports System
Imports System.IO

Module StreamWriterDemo
    Sub Main()
        Dim sw As New StreamWriter("StreamWriterDemo.txt", True)
        sw.WriteLine("You ran the StreamWriterDemo program on {0}", _
                     DateTime.Now)
        sw.Close()
    End Sub
End Module

File: win-prog/ProgWinCS/Files and Streams/StreamWriterDemo/StreamWriterDemo.cs
//-----------------------------------------------
// StreamWriterDemo.cs © 2001 by Charles Petzold
//-----------------------------------------------
using System;
using System.IO;

class StreamWriterDemo
{
     public static void Main()
     {
          StreamWriter sw = new StreamWriter("StreamWriterDemo.txt", true);

          sw.WriteLine("You ran the StreamWriterDemo program on {0}",
                       DateTime.Now);

          sw.Close();
     }
}
